usedHttp.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
+
+void makeRequest() async{
+    var response = await http.get('https://example.com/');
+    // If the http request is successful the statusCode will be 200
+    if (response.statusCode == 200){
+      var htmlToParse = response.body;
```

Comment:
Same here, please use final instead of var.

Reply:
I will make change it.